src/highscore.c

```diff
@@ -118,6 +118,13 @@ char *getUsersFullName()
     /* shorten to 20 chars */
     fullname[19] = 0;
 
+    /* SFont and TFont can only handle characters with values from 32d und 127d (ASCII) */
+    for (int i=0; i<=18; i++) {
+	if (fullname[i]>126 || fullname[i]<32) {
```

Comment:
I think you should use isascii() here instead.
This is just a bandaid though. This program is so old it doesn't know about UTF-8, and should be taught.

Reply:
or isprint()

Reply:
Thanks for the suggestion.
Well this is the patch - take it or leave it. It works fine in my fork. Feel free to fix it yourself, I opened an associated issue.

I am not sure I understand you - here https://github.com/frank-zago/xgalaga-sdl/issues/3#issuecomment-893103444, you said, the game was done and you only want bugfixes, but now you suggest to add UTF-8 support which is a bigger undertaking. Could you please make up your mind?

Reply:
Software should not crash, and the real fix here is to support utf8/unicode.
I wish you well with your fork. If it works I'll eventually archive that tree.